src/libraries/System.Runtime/tests/System.Runtime.Extensions.Tests/System/Random.cs

```diff
@@ -924,6 +924,81 @@ public static void GetItems_AllValuesInRange(int mode)
             }
         }
 
+        // -----
+
+        [Fact]
+        public static void GetString_ArgValidation()
+        {
+            Random random = new();
+            AssertExtensions.Throws<ArgumentException>("choices", () => random.GetString([], 42));
+            AssertExtensions.Throws<ArgumentOutOfRangeException>("length", () => random.GetString(['a'], -1));
+        }
+
+        [Fact]
+        public static void GetHexString_Array_ArgValidation()
+        {
+            Random random = new();
+            AssertExtensions.Throws<ArgumentOutOfRangeException>("length", () => random.GetHexString(-1, true));
+            AssertExtensions.Throws<ArgumentOutOfRangeException>("length", () => random.GetHexString(-2, false));
+        }
+
+        [Fact]
+        public static void GetString_ProducesExceptedStrings()
+        {
+            Random random = new Random(42);
+            Assert.Equal("", random.GetString("abcdefghijklmnopqrstuvwxyz", 0));
+            Assert.Equal("c", random.GetString("abcd", 1));
+            Assert.Equal("aaca", random.GetString("abcde", 4));
+            Assert.Equal("gsnetggnijgnavpkdcsvobsdxsnebi", random.GetString("abcdefghijklmnopqrstuvwxyz", 30));
+        }
+
+        [Fact]
+        public static void GetHexString_Array_ProducesExceptedStrings()
+        {
+            Random random = new Random(42);
+
+            Assert.Equal("", random.GetHexString(0));
+            Assert.Equal("A", random.GetHexString(1));
+            Assert.Equal("2282", random.GetHexString(4));
+            Assert.Equal("4B82C34856480D9621BD80B2EB8204", random.GetHexString(30));
+
+            Assert.Equal("", random.GetHexString(0, false));
+            Assert.Equal("9", random.GetHexString(1, false));
+            Assert.Equal("D08C", random.GetHexString(4, false));
+            Assert.Equal("B6200CA13C4A209806BA30541B170C", random.GetHexString(30, false));
+
+            Assert.Equal("", random.GetHexString(0, true));
+            Assert.Equal("7", random.GetHexString(1, true));
+            Assert.Equal("870d", random.GetHexString(4, true));
+            Assert.Equal("001a0ee7690526c864e9b0ef5b2175", random.GetHexString(30, true));
+        }
+
+        [Fact]
+        public static void GetHexString_Span_ProducesExceptedItems()
```

Comment:
More of the same typo.  Hopefully this suggestion only has two renames in it.

```suggestion
        public static void GetHexString_Array_ProducesExpectedStrings()
        {
            Random random = new Random(42);

            Assert.Equal("", random.GetHexString(0));
            Assert.Equal("A", random.GetHexString(1));
            Assert.Equal("2282", random.GetHexString(4));
            Assert.Equal("4B82C34856480D9621BD80B2EB8204", random.GetHexString(30));

            Assert.Equal("", random.GetHexString(0, false));
            Assert.Equal("9", random.GetHexString(1, false));
            Assert.Equal("D08C", random.GetHexString(4, false));
            Assert.Equal("B6200CA13C4A209806BA30541B170C", random.GetHexString(30, false));

            Assert.Equal("", random.GetHexString(0, true));
            Assert.Equal("7", random.GetHexString(1, true));
            Assert.Equal("870d", random.GetHexString(4, true));
            Assert.Equal("001a0ee7690526c864e9b0ef5b2175", random.GetHexString(30, true));
        }

        [Fact]
        public static void GetHexString_Span_ProducesExpectedItems()
```